BasiliskII/src/Unix/timer_unix.cpp

```diff
@@ -57,14 +57,14 @@ static inline void mach_current_time(tm_time_t &t) {
 void Microseconds(uint32 &hi, uint32 &lo)
 {
 	D(bug("Microseconds\n"));
-#if defined(HAVE_CLOCK_GETTIME)
-	struct timespec t;
-	clock_gettime(CLOCK_REALTIME, &t);
-	uint64 tl = (uint64)t.tv_sec * 1000000 + t.tv_nsec / 1000;
-#elif defined(__MACH__)
+#if defined(PRECISE_TIMING_MACH)
```

Comment:
Should the #if defined(__MACH__) at the top of the file be changed to PRECISE_TIMING_MACH too?